module/VuFind/src/VuFind/Auth/Manager.php

```diff
@@ -516,19 +523,19 @@ public function logout($url, $destroy = true)
      *
      * @return bool
      */
-    public function userHasLoggedOut()
+    public function userHasLoggedOut(): bool
     {
         return (bool)$this->cookieManager->get('loggedOut');
     }
 
     /**
-     * Checks whether the user is logged in.
+     * Legacy method to check whether the user is logged in.
```

Comment:
This method is already slated for removal in 11.0 via [VUFIND-1655](https://openlibraryfoundation.atlassian.net/issues/VUFIND-1655). Maybe it's worth just deleting it rather than revising the comment -- it's going to go away soon one way or another.